.github/actions/init-local/action.yml

```diff
@@ -75,6 +75,12 @@ runs:
                 size: 1Gi
         EOM
 
+        # Change jupyterhub image for PR testing purposes
+        cat >> '${{ steps.metadata.outputs.config }}' <<- EOM
+        default_images:
+          jupyterhub: quay.io/nebari/nebari-jupyterhub:jhub-fancy-profiles
+        EOM
+
```

Comment:
@Adam-D-Lewis , I think this was not suposed to be committed as part of this PR, is that right?

Reply:
NVM, just saw your comment above.